pallets/ddc-verification/src/lib.rs

```diff
@@ -1393,6 +1481,41 @@ pub mod pallet {
 			)))
 		}
 
+		pub(crate) fn fetch_minimum_sub_aggregates(
```

Comment:
why it's named 'minimum sub aggregates'? didn't find any logic with selecting min...